packages/grafana-data/src/types/teams.ts

```diff
@@ -0,0 +1,58 @@
+export interface Team {
+  id: number; // TODO switch to UUID
```

Comment:
Team apis should support both using id and uid. If we can we should only use uid and remove id

Reply:
Is this something we want to do in this PR, I'm trying to move this component with minimal impact. Moving over from ids to uids sounds like we'd need to do a migration as well?

Reply:
No we don't need a migration. That was done like 1 year ago and we added support for using it through the api like 1 month ago. 

Especially because we now would expose these to plugins we probably want to do these fixes before so we don't have to break it. I can look into this tomorrow and make a pr to fix it. Then you can port it after?